src/standalone/Vegetation/photosynthesis.jl

```diff
@@ -1,4 +1,33 @@
-export FarquharParameters, FarquharModel, C3, C4
+export SIFParameters, FarquharParameters, FarquharModel, C3, C4
+
+abstract type AbstractPhotosynthesisModel{FT} <: AbstractCanopyComponent{FT} end
+
+"""
+    SIFParameters{FT<:AbstractFloat}
+
+The required parameters for the SIF parameterisation [give citation].
+$(DocStringExtensions.FIELDS)
+"""
+@kwdef struct SIFParameters{FT <: AbstractFloat}
+    ""
```

Comment:
The rate coefficient for florescence, unitless. Tol et al 2014 via Lee et al 2015